source/upgrade.txt

```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
+|compass-short| automatically updates the application four hours after
+MongoDB releases new versions. The update does not require your explicit
+confirmation. After your machine finishes the update, |compass-short|
+shows a toast suggesting that you restart the application to apply the
```

Comment:
```suggestion
shows a notification suggesting that you restart the application to apply the
```

I don't know if users know what a toast is, tbh.

Reply:
Changed to "notification"!